java-test.java

```diff
@@ -20,35 +19,35 @@ public String getName() {
     }
 
     public double averageScore() {
-        if (scores.isEmpty()) {
+        System.out.println("scores.isEmpty() " + scores.isEmpty());
+        if (scores.isEmpty) {
```

Comment:
suggested comment:  System.out.println("scores.isEmpty() " + scores.isEmpty()); in 22